### PR TITLE
Add CircuitPython code for CPX compass project

### PR DESCRIPTION
**Pull request by dastels**

### CODE REVIEW

File: CPX_Compass/code.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    raw_maxes[1] = -1000.0
+    calibrate(True)
+
+
+    print("Calibration results")
+    print("Update the corresponding lines near the top of the code\n")
+    print("raw_mins = [{0}, {1}]".format(raw_mins[0], raw_mins[1]))
+    print("raw_maxes = [{0}, {1}]".format(raw_maxes[0], raw_maxes[1]))
+    while True:
+        pass
```
Comment (tannewt) [inline CPX_Compass/code.py]: Why busy wait here?

Comment (TheKitty) [reply CPX_Compass/code.py]: Dave, check please

Comment (dastels) [reply CPX_Compass/code.py]: It’s the calibration mode: Calibrate, output the lines to update the code with, and stall. Once the code is updated with the calibration results this code isn’t run and the compass works as a compass. Starting with button A pressed to  drop into this mode as well to,generate new calibration lines.

Comment (dastels) [reply CPX_Compass/code.py]: Alternatively, is there a way to exit to the repl immediately?

Comment (tannewt) [reply CPX_Compass/code.py]: try `sys.exit()`. That's how python would do it and I think it works in CircuitPython too.

**Comment by dastels** [comment]
Done, tested, pushed, passed.  Ready to merge.

I don't seem to be getting notifications from github... something to look into.

**Comment by ladyada** [comment]
thank you! mergin' :)

**Pull request merged**